### ISSUE TITLE
Cask formulas were not requested/updated by brewlet. Brewlet doesn't use json=v2.

### ISSUE THREAD

**Issue by snetwork-dev** [zkokaja/Brewlet#27]
Currently cask formulas were not requested nor updated by brewlet.
It is probably because `--json=v2` is not used.

See also `man brew`:
```
      --json                       Print a JSON representation. Currently the
                                   default value for version is v1 for
                                   formula. For formula and cask use
                                   v2. See the docs for examples of using
                                   the JSON output:
                                   https://docs.brew.sh/Querying-Brew
```
								   
The following patch should fix the problem:
```
@@ -235,7 +235,7 @@
         let packageItem = self.statusMenu.item(withTag: self.name2tag["packages"]!)!
         packageItem.isEnabled = false
         
-        run_command(arguments: ["info", "--json", "--installed"]) { (_, data: Data) in
+        run_command(arguments: ["info", "--json=v2", "--installed"]) { (_, data: Data) in
             
             // Determine which packages to include
             let includeDependencies = self.userDefaults.bool(forKey: "includeDependencies")
```

**Comment by cmpadden** [comment]
Hey @snetwork-dev, thanks for pointing out `--json=v2`.

I took a quick look at this and I don't think it will be as simple as replacing `--json` with `--json=v2`, as the response schema differs in v2. Because of that, the code that deserializes the response would have to be updated.

https://github.com/zkokaja/Brewlet/blob/6feaef146e667fb9e990dd77825e23e4144e9821/Brewlet/Package.swift#L73-L78

Additionally, I think that the code that triggers the `brew upgrade` command would have to be updated to run `brew upgrade --cask` for the scenario in which a package is a cask.

https://github.com/zkokaja/Brewlet/blob/6feaef146e667fb9e990dd77825e23e4144e9821/Brewlet/AppDelegate.swift#L350

**Comment by zkokaja** [comment]
Thanks for the input. I implemented this changed and commented on #19.

Note, `brew upgrade` will upgrade casks as is currently implemented, while `brew upgrade --cask` will __only__ upgrade casks and not formulae.

**Comment by zkokaja** [comment]
Implemented in v1.6
